frontend/src/SearchBar/SearchBar.jsx

```diff
@@ -0,0 +1,23 @@
+import { useState, useEffect } from 'react';
+import "./SearchBar.css"
+
+
+
+function SearchBar (){
+    const [search, setSearch] = useState("");
+
+    function handleSearchChange(event) {
+        setSearch(event.target.value);
+        setUrl(); // API KEY
+    }
+
+    return(
+    <form className="search-bar">
+        <input id="search" type="text" value={search} className="search-input" placeholder="Search..." onChange={e => handleSearchChange(e)}/>
+        <button id="clear" className="clear-results">Clear</button>
```

Comment:
shouldn't the search button named search button.

Reply:
I was going to implement searching with the flutter because I've done it in all my other projects. So having a search button wouldn't be needed because it'll already be updating and displaying the results as the input changes.